src/material/form-field/form-field.html

```diff
@@ -16,12 +16,12 @@
   -->
   <label matFormFieldFloatingLabel
          [floating]="_shouldLabelFloat()"
+         [monitorResize]="appearance === 'outline'"
```

Comment:
There's already a `_hasOutline` helper that checks for `appearance === 'outline'`.